includes/troubleshooting/flutter/enable_multi_view.mdx

```diff
@@ -0,0 +1,216 @@
+          child: const MyApp(),
+        ),
+      ),
+    ),
+    exampleDsn,
+  );
+}
+
+Future<void> setupSentry(
+...
+}) async {
+  await SentryFlutter.init(
+    (options) {
+      ...
+      final integration = options.integrations
+          .firstWhere((element) => element is WidgetsBindingIntegration);
+      options.removeIntegration(integration);
+    },
+    // Init your App.
+    appRunner: appRunner,
+  );
+}
+
+class MyApp extends StatefulWidget {
+  const MyApp({super.key});
+
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  @override
+  Widget build(BuildContext context) {
+    return feedback.BetterFeedback(
+      child: ChangeNotifierProvider<ThemeProvider>(
+        create: (_) => ThemeProvider(),
+        child: Builder(
+          builder: (context) => MaterialApp(
+            navigatorObservers: [
+              SentryNavigatorObserver(),
+            ],
+            theme: Provider.of<ThemeProvider>(context).theme,
+            home: const MainScaffold(),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+...
+
+
+class MainScaffold extends StatelessWidget {
+  ...
+
+  @override
+  Widget build(BuildContext context) {
+    ...
+    return Scaffold(
+      appBar: AppBar(
+        title:
+            Text('Sentry Flutter Example (ViewId:${View.of(context).viewId})'),
+        actions: [
+          ...
+        ],
+      ),
+      body: SingleChildScrollView(
+        ...
+      ),
+    );
+  }
+
+  ...
+}
+
+
+```
+
+```js {filename:flutter_bootstrap.js}{tabTitle: flutter_bootstrap.js}
+// flutter_bootstrap.js
+{{ flutter_js; }}
+{{ flutter_build_config; }}
+
+_flutter.loader.load({
+  onEntrypointLoaded: async function onEntrypointLoaded(engineInitializer) {
+    let engine = await engineInitializer.initializeEngine({
+      multiViewEnabled: true, // Enables embedded mode.
+    });
+    let app = await engine.runApp();
+    // Make this `app` object available to your JS app.
+    app.addView({ hostElement: document.querySelector("#left") });
+    app.addView({ hostElement: document.querySelector("#right") });
+  },
+});
+```
+
+```html {filename:index.html}{tabTitle: index.html}
+<!doctype html>
+<html>
+  ...
+  <body>
+    <div style="width: 100%; height: 100%; position: absolute;">
+      <div id="left" style="width: 50%; height: 100%; float: left;"></div>
+      <div id="right" style="width: 50%; height: 100%; float: left;"></div>
+    </div>
+    <!-- This script installs service_worker.js to provide PWA functionality to
+       application. For more information, see:
+       https://developers.google.com/web/fundamentals/primers/service-workers -->
+    <script>
+      if ("serviceWorker" in navigator) {
+        window.addEventListener("load", function () {
+          navigator.serviceWorker.register("flutter_service_worker.js");
+        });
+      }
+    </script>
+    <script src="main.dart.js" type="application/javascript"></script>
```

Comment:
`flutter_bootstrap.js` will add `main.dart.js` and register `flutter_service_worker`, so these two script tags could be removed

Reply:
thanks for the info.